frappe/public/js/frappe/form/layout.js

```diff
@@ -111,6 +112,8 @@ frappe.ui.form.Layout = class Layout {
 			}
 			this.message.removeClass("hidden").addClass(this.message_color);
 			$(html).appendTo(this.message);
+			close_message.appendTo(this.message);
+			close_message.on("click", () => this.message.empty().addClass("hidden"));
```

Comment:
This will come back on next page load... should we remember preference? (at least in same browser)

Reply:
It is not simple since we do not maintain these messages they can come from anywhere.